include/meta/meta.hpp

```diff
@@ -3162,12 +3173,15 @@ namespace meta
         /// \f$ O(log(N)) \f$.
         /// \ingroup integral
         template <typename T, T N>
-        using make_integer_sequence =
+        using make_integer_sequence = _t<
             _t<detail::coerce_indices_<T, 0, make_index_sequence<static_cast<std::size_t>(N)>>>;
```

Comment:
lol, the problem, is the content in meta.hpp is sometimes so advanced shi**,  I always stumble across stuff where I think "wtf, whats happening here :-)" which are not at all easy to understand but cool, I've seen this `t_<` -> advanced technique? hää? :-)  thx for the headache relieve :-)